components/tools/tools-display.tsx

```
@@ -0,0 +1,177 @@
+        const isExpanded = expandedCategory === category;
+        
+        return (
+          <div key={category} className="w-full">
+            <button 
+              onClick={() => setExpandedCategory(isExpanded ? null : category as ToolCategory)}
+              className={cx(
+                'w-full rounded-lg p-3 text-left font-medium transition-all',
+                bgColor,
+                textColor
+              )}
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)